tests/drivers/entropy/api/boards/google_dragonclaw.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2024 Google Inc
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&rng {
+	status = "okay";
+};
```

Comment:
why not enabling it in the board dts file? that's what most stm32 boards do

Reply:
Actually, I don't see a reason why not, I'll change that.

Reply:
Done.